runtimes/runtimes/lsp/router/lspRouter.ts

```diff
@@ -32,11 +34,34 @@ export class LspRouter {
         lspConnection.onRequest(getConfigurationFromServerRequestType, this.handleGetConfigurationFromServer)
     }
 
+    public get clientSupportsShowNotification() {
+        return (
+            this.clientInitializeParams?.initializationOptions?.aws.awsClientCapabilities?.window?.showNotification ??
+            false
+        )
+    }
+
     initialize = async (
         params: InitializeParams,
         token: CancellationToken
     ): Promise<InitializeResult | ResponseError<InitializeError>> => {
         this.clientInitializeParams = params
+
+        let responsesList = await Promise.all(this.servers.map(s => s.initialize(params, token)))
+        responsesList = responsesList.filter(r => r != undefined)
+        const errors = responsesList.find(el => el instanceof ResponseError)
+        if (errors) {
+            return errors as ResponseError<InitializeError>
+        }
+        const dupServerNames = findDuplicates(responsesList.map(r => (r as PartialInitializeResult).serverInfo?.name))
```

Comment:
`findDuplicates` perform simple direct value comparison, is this final implementation or will it change?

Reply:
We can look to extend it if dedup of objects is ever needed. Now it's working for primitives and objects by ref.